Web/starter_project_vanguard/components/team/HeroSection.tsx

```diff
@@ -0,0 +1,24 @@
+import Image from "next/image"
+
+const HeroSection = () => {
```

Comment:
Please define a component with a syntac like React.FC (or whatever is the best way for nextJS) to clarify what is what